src/Sqs/Tests/Integration/SqsClientTest.php

```diff
@@ -14,167 +12,104 @@
 use AsyncAws\Sqs\Input\PurgeQueueRequest;
 use AsyncAws\Sqs\Input\ReceiveMessageRequest;
 use AsyncAws\Sqs\Input\SendMessageRequest;
-use PHPUnit\Framework\TestCase;
+use GetClient;
 
-class SqsClientTest extends TestCase
+class SqsClientTest extends \AsyncAws\CodeGenerator\Generator\TestCase
 {
-    use GetClient;
-
-    public function testChangeMessageVisibility()
+    public function testChangeMessageVisibility(): void
     {
-        $sqs = $this->getClient();
+        self::markTestSkipped('Not implemented');
 
-        $sqs->createQueue(['QueueName' => 'foo'])->resolve();
-        $fooQueueUrl = $sqs->getQueueUrl(['QueueName' => 'foo'])->getQueueUrl();
-        $sqs->purgeQueue(['QueueUrl' => $fooQueueUrl])->resolve();
```

Comment:
Make sure we dont remove the current tests =)

Reply:
fixed